### ISSUE TITLE
New updated forks of Vxkex that may help to keep vital ecological support for windows 7 os 32 bits please ! 

### ISSUE THREAD

**Issue by trimechee** [Alex313031/thorium-win7#126]
Hello, the great project Vxkex that allows windows 7 os to use windows 10/11 os softwares has been removed from github, some say microsoft fears this project so they banned it from github,...

So today I find new rare projects of Vxkex that add updates features, fix bugs, add support for new softwares to windows 7 os !! may be these new updated forks can help the great Thorium Legacy to keep vital ecological support of our windows 7 os 32 bits please !

and obviously I think wndows 7 os users will be happy to install Vxkex which is very easy to use and without risk for the os! Thank's !

https://github.com/i486/VxKex/issues/36 (YuZhouRen86 fork updated)

https://github.com/Aqua-Vision/MgKex (fork updated)

https://github.com/i486/VxKex (clone of original project)

extended kernel by Mr dotexe1337
http://www.dotexe.cf/extended/
https://web.archive.org/web/20241006001741/https://www.dotexe.cf/extended/
https://github.com/kernel-extenders/nt61

windows 7 os improve usuablility 32 bits (use with caution, prudence)
https://github.com/winrevive/win7-sp2
https://sp2.winrevive.xyz/

**Comment by hubstedt** [comment]
Thank you for bringing this to attention! I was unaware of further development to VxKex.

**Comment by trimechee** [comment]
Thank's :)

**Comment by trimechee** [comment]
@hubstedt , Hi, i find new tools this night ! : 


extended kernel by Mr dotexe1337
http://www.dotexe.cf/extended/
https://web.archive.org/web/20241006001741/https://www.dotexe.cf/extended/
https://github.com/kernel-extenders/nt61

windows 7 os improve usuablility 32 bits (use with caution, prudence, can break system)
https://github.com/winrevive/win7-sp2
https://sp2.winrevive.xyz/

**Comment by Half-Modern** [comment]
> windows 7 os improve usuablility 32 bits (use with caution, prudence) https://github.com/winrevive/win7-sp2 https://sp2.winrevive.xyz/
>
It doesn't even support 32-bit version of Windows 7.
Also there are reports that VxKex is not working properly on W7 32-bit which means you are limited to pre-hacked to work on 7 natively stuff.

**Comment by trimechee** [comment]
Interesting ! Thank you very much for your precious advises Mr Half-Modern :) 💯

**Comment by Templayer** [comment]
> > windows 7 os improve usuablility 32 bits (use with caution, prudence) https://github.com/winrevive/win7-sp2 https://sp2.winrevive.xyz/
> 
> It doesn't even support 32-bit version of Windows 7. Also there are reports that VxKex is not working properly on W7 32-bit which means you are limited to pre-hacked to work on 7 natively stuff.

There is a new version that should be working on Win7 32-bit: https://github.com/i486/VxKex/releases/tag/Version1.1.2.1428

**Comment by trimechee** [comment]
Great!! i see the new version of Vxkex fix specefic bugs i indows 7 os 32 bits, thank you very much Mr @Templayer for this important information :) 💯:


Please Mr @Templayer and Mr @Half-Modern :

this is the first time i will install Vxkex but there is things i can't understand anh what to do exactly to use Vxkex :


so please i need help, there is one thing i can't understand, in the description of Vxkex: "After installation, usage is easy: just right click on a program (.exe or .msi), open the Properties dialog, and select the VxKex tab. Then, check the check box which says "Enable VxKex for this program", and try to run the program."

for example i like to install official version of firefox or chromium in my windows 7 OS 32 bits,, but firefox and chromium are not compatible with windows 7 os 32 bits, so logically i can't install firefox and chromium and mpc qt and therefore i can't use Vxkex? i won't have error message when installing firefox, mpc qt ....? missing api or ddl? I specify I have never tried Vxkex but I prefer to ask these questions before starting the adventure to download software then nothing will work maybe....

ahh I just thought, I thought we have to install Vxkex and firefox, but we have to install Vxkex then right click on the firefox exe? but how to get the firefox exe without installation? I don't understand anything anymore, I'm in the fog, please I need help, thank's!


i would like also to install Coc Coc browser and use it with Vxkex, because windws 7 version of Coc Coc browser in no more updated so i need to use te version Coc Coc browser for windows 10, because Coc coc browser has rare awesome internal media donwloader that supports donwload from youtbe, embedded hls videos.....,

so in the site of coc coc browser, i will download only little .exe installer size 2 mo that will install the browser online, how add Vxkex to coco coc browser please? i need to donwoad offline version of Vxkex then enable Vxkex please? has

**Comment by Templayer** [comment]
> Great!! i see the new version of Vxkex fix specefic bugs i indows 7 os 32 bits, thank you very much Mr @Templayer for this important information :) 💯:
> 
> Please Mr @Templayer and Mr @Half-Modern :
> 
> this is the first time i will install Vxkex but there is things i can't understand anh what to do exactly to use Vxkex :
> 
> so please i need help, there is one thing i can't understand, in the description of Vxkex: "After installation, usage is easy: just right click on a program (.exe or .msi), open the Properties dialog, and select the VxKex tab. Then, check the check box which says "Enable VxKex for this program", and try to run the program."
> 
> for example i like to install official version of firefox or chromium in my windows 7 OS 32 bits,, but firefox and chromium are not compatible with windows 7 os 32 bits, so logically i can't install firefox and chromium and mpc qt and therefore i can't use Vxkex? i won't have error message when installing firefox, mpc qt ....? missing api or ddl? I specify I have never tried Vxkex but I prefer to ask these questions before starting the adventure to download software then nothing will work maybe....
> 
> ahh I just thought, I thought we have to install Vxkex and firefox, but we have to install Vxkex then right click on the firefox exe? but how to get the firefox exe without installation? I don't understand anything anymore, I'm in the fog, please I need help, thank's!
> 
> i would like also to install Coc Coc browser and use it with Vxkex, because windws 7 version of Coc Coc browser in no more updated so i need to use te version Coc Coc browser for windows 10, because Coc coc browser has rare awesome internal media donwloader that supports donwload from youtbe, embedded hls videos.....,
> 
> so in the site of coc coc browser, i will download only little .exe installer size 2 mo that will install the browser online, how add Vxkex to coco coc browser please? i need to donwoad offline version of Vxkex then enable Vxkex please? has

I wouldn't quite recommend using browsers with VxKex, as they can break anytime.

For browsers under Win7 (64-bit for me, but Thorium Legacy has 32-bit versions as well, I think), I am using:

* Chromium-based ones: Thorium Legacy (I am actually paying for this one, but banking doesn't work) and Supermium
* Firefox-based ones: Waterfox and r3dfox (these might be 64-bit only, though - haven't checked)
* Goanna-based ones: Pale Moon and Basilisk (there are however quite obsolete and some pages break)

Now for VxKex - right-click the installer or the installed exe file, go to properties and you should see a new tab:

![image](https://github.com/user-attachments/assets/81f852ca-4765-4b1a-aa29-76c9690eef3d)

As you see, quite simple. VxKex enables me to run qt6-based applications, like the newer versions of HandBrake and qbittorrent. qt6 doesn't support or work by default under Win7, so this is a boon!

If an application has problems working, try out the various advanced options in the tab. I had to disable enabling VxKex for child processes created by the main app process for something, but I forgot for what.

**Comment by trimechee** [comment]
Ohhh awesome great! thank you so much Mr @Templayer  from these important and precious explanations and complaints!


for palemoon and basilisk browsers, i already installed them and they are must have vital for every windows 7 os and even for windows 11, modern os, mac....

 PALEMOON AND BASILISK are updated regularly, security features, new features enhancement are added regularly, see the changelogs, , they are light no electrolysis so much light that other browser, smooth playback, and here 2 updated adsblockers manifest v2 based on ublock origin, firefox or brave may abandon the v2 manifest in the future for reasons of compatibility so palemonn and basilisk are the last chance for adsblockers manifest v2:


palemoon basilisk browsers adblockers pale moon :

AstroSkipper
https://msfn.org/board/topic/183923-extensions-and-custom-buttons-for-uxp-browsers-corrections-modifications-adjustments-and-special-recommendations/page/16/#comment-1258602
https://msfn.org/board/topic/183923-extensions-and-custom-buttons-for-uxp-browsers-corrections-modifications-adjustments-and-special-recommendations/page/7/#comment-1257610
UCyborg
https://github.com/UCyborg/uBlock-for-firefox-legacy
https://forum.palemoon.org/viewtopic.php?f=46&t=31050&sid=0bfecfb0bf859bc6cd28478017767edb
https://projectfresco.github.io/addons/get?addon=ublock-origin
https://forum.palemoon.org/viewtopic.php?f=46&t=31050&sid=0bfecfb0bf859bc6cd28478017767edb

https://msfn.org/board/topic/183923-extensions-and-custom-buttons-for-uxp-browsers-corrections-modifications-adjustments-and-special-recommendations/

**Comment by Templayer** [comment]
> firefox or brave may abandon the v2 manifest in the future for reasons of compatibility so palemonn and basilisk are the last chance for adsblockers manifest v2

That statement is incorrect. The Thorium author is dedicated to keep V2 working in Thorium. I.e. Pale Moon and Basilisk aren't the last chance for ad blockers. Given his GUI reverts, I am inclined to believe him. He is skilled enough to revert its removal.

Pale Moon and Basilisk are old and a lot of things break. Twitch was broken for me, Facebook had a broken chat (I cannot write in it until I copy-paste something inside). It is not their fault, however - they are forks of old Gecko and it is hard for the few devs they have to keep up.

Also, check out MyPal - the Quantum version specifically. Works on WinXP 32-bit too.

Please note that I will stop using any browser that stops having adblocker just because the corporate fucktards at Google said so. I am not interested in any milfs in my area, thank you. Even if it means using an obsolete browser (i.e. Pale Moon) instead.

**Comment by trimechee** [comment]
Yes, I don't use palmeoon and basilisk on sites like twitter, twithch....I just use it to watch streaming videos because they are lighter compared to other web browsers that lag.....

Yes Thorium and Supermium say they will keep manifest v2, good for them!

concerning firefox and brave (and catsxp which seems to be based on brave), they said they will support manifest v2 as long as it is possible, and as long as there are developers who support manifest v2 concerning brave ....

otherwise Mr, here are some alternatives if adsblockers become ineffective for one reason or another:


https://nxfilter.org/p4/
https://github.com/filecxx/WebOptimizer
http://www.filecxx.com/en_US/index.html
https://www.ghostery.com/
https://eblocker.org/en/
https://safing.io/
https://github.com/safing/portmaster
https://www.sordum.org/13075/url-disabler-v1-2/
https://one.one.one.one/
https://gitlab.com/Pecha/webfilter


https://sqrx.com/


otherwise I tried ublock lite manifest v3 in chrome windows 7:

https://github.com/Blaukovitch/GOOGLE_CHROME_Windows_7

i am pleasantly surprised, i already seen that the rules are limited to 30k, so I disabled the malware blocking blocklist, adguard regional french and I chose the full mode, and ublock origin lite blocks ads on my favorite streaming site!

**Comment by trimechee** [comment]
it seems adblockers manifest v3 have adapted to the constraints of manfest v3 and have adapted adaptive rules which change according to the sites and despite the technical limitation, adblockers seem to effectively block ads and it says it uses less resources than adblocker manifest v2

**Comment by Templayer** [comment]
> it seems adblockers manifest v3 have adapted to the constraints of manfest v3 and have adapted adaptive rules which change according to the sites and despite the technical limitation, adblockers seem to effectively block ads and it says it uses less resources than adblocker manifest v2

OK, that is good news. And a slap in the face of Google.

Even though a part of me wishes that it wouldn't be so, as it would cause mainline unforked Chromium to go the way of the dodo... er... I mean... Internet Explorer.

I still remember when Internet Explorer was the most used browser on the planet. It languished, refusing to develop new features and forcing everyone to adapt to it instead.

How well has that worked out?

Google, however, is too dumb not to repeat the mistakes others made in the past.

Instead, it also produces new ones, like forcing everybody into user interfaces that literally nobody asked for that are butt-fucking-ugly.

**Comment by trimechee** [comment]
Yes, google chrome should be more attentive to many developers who refused the manifest v3 which is too liberticidal, otherwise it could happen to chrome what happened to internet explorer, mac and ios are becoming more and more popular and safari is a serious competitor, chrome must be more flexible, for example allow the installation of extensions outside the store, I tried some time ago and even with the developer mode, chrome refuses extensions outside the store for example extension which allows you to download youtube, maybe it has changed since then....

**Comment by Templayer** [comment]
> Yes, google chrome should be more attentive to many developers who refused the manifest v3 which is too liberticidal, otherwise it could happen to chrome what happened to internet explorer, mac and ios are becoming more and more popular and safari is a serious competitor, chrome must be more flexible, for example allow the installation of extensions outside the store, I tried some time ago and even with the developer mode, chrome refuses extensions outside the store for example extension which allows you to download youtube, maybe it has changed since then....

I stopped using the youtube downloading extension when I started using JDownloader. Run on Linux. Runs on WinXP. Runs on basically anything (at least the Java 8 version does, as that version of Java runtime is ancient). And it has youtube plugin that is already included when you install it, and it auto-updates when youtube causes changes that break it.

![image](https://github.com/user-attachments/assets/6164926e-4103-41c5-8c19-adbfa0eb3dec)

A superior all-in-one (except for Twitch, strangely) downloading software suite.

**Comment by Half-Modern** [comment]
@Templayer That's good news for 32-bit users. Bit strange to see the author of the [video](https://www.youtube.com/watch?v=Akd30-18Wlo) I've watched a long time ago though 😅 
@trimechee For "lightweight" web experience, you can try changing a browser's user agent to a mobile version and keep it like that and use that browser for basic browsing.
For Youtube on Firefox (much slower than Chromium), I use these extensions:
[Youtube mobile for Desktop (Changes YT to mobile version with enhancements)](https://addons.mozilla.org/en-US/firefox/addon/youtube-mobile-for-desktop/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search)
[Youtube Mobile Fix (Adds keyboard shortcuts for playback from the desktop back)](https://addons.mozilla.org/en-US/firefox/addon/mobile-youtube-fix/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search)

**Comment by Templayer** [comment]
> @Templayer That's good news for 32-bit users. Bit strange to see the author of the [video](https://www.youtube.com/watch?v=Akd30-18Wlo)

Yeah, I still have that Win XP 32-bit multi-booted with Win7 64-bit on my main desktop PC.

I did some more videos, like for example hacking the driver of a modern pricey external sound card to work under WinXP 32-bit. This enables me to use my two XLR-connected studio microphones under native nonvirtualized WinXP. They are crisp and clear!

It is among many different things I do. I really like my retro stuff. ¯\_(ツ)_/¯

**Comment by trimechee** [comment]
Ohh very interesting and informative! Thanks @Templayer @Half-Modern  for sharing these tips and important information and softwares :)

**Comment by Templayer** [comment]
> Ohh very interesting and informative! Thanks @Templayer @Half-Modern for sharing these tips and important information and softwares :)

No problem. Glad to be helpful, as always.

**Comment by trimechee** [comment]
💯 🥇 👍

**Comment by TotallyNotTemplayerWinkWink** [comment]
Well, I finally got a response on the impersonation report!

![image](https://github.com/user-attachments/assets/34429ab9-6320-4359-ad15-41562cce0bd3)

Welp. They didn't find anything wrong with the fake "vxiiduu", but I am a bad apple.

Thanks, Microsoft! Competent as ever!

My comments were deleted. Thanks to this "Sin-Shadow-Fox" over here: https://github.com/i486/VxKex/issues/41

Fret not, I have backups of backups... thankfully, I only use github for making comments, I wouldn't dare using Microsoft's newest "Skype" for anything.

![image](https://github.com/user-attachments/assets/ab6fd8e5-cc3b-4838-95e5-8a80897a1ea8)
![image](https://github.com/user-attachments/assets/6fe1a954-f22c-4b59-8c51-8f5c1e90de8d)
![image](https://github.com/user-attachments/assets/eb32bf96-c93d-418d-a0b3-7f9cef8a2877)
![image](https://github.com/user-attachments/assets/b8db9222-6fa2-4b68-a690-f9435faaaa5d)
![image](https://github.com/user-attachments/assets/dffb2594-febe-4aeb-a695-d97af8ad64ff)
![image](https://github.com/user-attachments/assets/966da2c9-31ba-4b22-8feb-42f19eca3707)
![image](https://github.com/user-attachments/assets/4ef6bf07-f392-4877-af9c-4de35af44354)
![image](https://github.com/user-attachments/assets/dd687c76-15eb-4f68-8dcb-7cfe5272f93c)
![image](https://github.com/user-attachments/assets/36219e0f-1218-47a3-8884-ccc554096fad)

I'll be over at GitLab, the superior Git implementation. Is it perfect? Hardly. But it is out of M$'s grasp... and that is enough for me. :3

**Comment by Half-Modern** [comment]
That's very poor of Github. I'd avoid opening any repos anyway due to their forced 2FA (which caused an account to be locked down). Their administration isn't even close to good from what I see too.
